flytekit/clis/sdk_in_container/package.py

```diff
@@ -0,0 +1,231 @@
+import os
+import sys
+import tarfile
+import tempfile
+import typing
+from collections import OrderedDict
+
+import click
+from flyteidl.admin import launch_plan_pb2, task_pb2, workflow_pb2
+
+from flytekit import LaunchPlan
+from flytekit.clis.sdk_in_container import constants, serialize
+from flytekit.common import translator
+from flytekit.configuration import internal
+from flytekit.core import context_manager
+from flytekit.core.base_task import PythonTask
+from flytekit.core.context_manager import ImageConfig, look_up_image_info
+from flytekit.core.workflow import WorkflowBase
+from flytekit.tools import fast_registration, module_loader
+
+_DEFAULT_IMAGE_NAME = "default"
+_DEFAULT_RUNTIME_PYTHON_INTERPRETER = "/opt/venv/bin/python3"
+
+
+def validate_image(ctx: typing.Any, param: str, values: tuple) -> ImageConfig:
+    """
+    Validates the image to match the standard format. Also validates that only one default image
+    is provided. a default image, is one that is specified as
+      default=img or just img. All other images should be provided with a name, in the format
+      name=img
+    """
+    default_image = None
+    images = []
+    for v in values:
+        if "=" in v:
+            vals = v.split("=")
```

Comment:
```suggestion
            vals = v.split("=", maxsplit=1)
```

Reply:
Are you suggesting that if  there are more than 1 `=` we should just take it and let the image library figure that out?

Reply:
done